web/docs/language/features.md

```diff
@@ -602,6 +602,193 @@ try {
 }
 ```
 
+### Automatic CRUD
+
+Wasp supports automatic action and query generation based of your Prisma models. It will generate the Create, Read (Read All), Update and Delete actions and queries for each model. You can then use them in your client code.
+
+The operations Wasp supports are:
+- **getAll** - returns all entities
+- **get** - returns one entity by id field (field marked with `@id` in Prisma schema)
+- **create** - creates a new entity
+- **update** - updates an existing entity
+- **delete** - deletes an existing entity
+
+#### CRUD declaration
+
+The CRUD declaration works on top of an existing entity declaration. It is declared as follows:
+
+```wasp title="main.wasp"
+crud Tasks { // crud name here is "Tasks"
+  entity: Task,
+  operations: {
+    getAll: {
+      isPublic: true, // optional, defaults to false
+    },
+    get: {},
+    create: {
+      overrideFn: import { createTask } from "@server/tasks.js", // optional
+    },
+    update: {},
+  },
+}
+```
+ 
+It has the following fields:
+- `entity: Entity` - the entity to which the CRUD operations will be applied.
+- `operations: { [operationName]: CrudOperationOptions }` - the operations to be generated. The key is the name of the operation, and the value is the operation configuration.
+  - The possible values for `operationName` are:
+    - `getAll`
+    - `get`
+    - `create`
+    - `update`
+    - `delete`
+  - `CrudOperationOptions` can have the following fields:
+    - `isPublic: bool` - Whether the operation is public or not. If it is public, it will be available to all users. If it is not public, it will be available only to authenticated users. Defaults to `false`.
+    - `overrideFn: ServerImport` - The import statement of the optional override implementation in Node.js.
+
+#### Defining the overrides
+
+Like with actions and queries, you define the implementation in a Javascript/Typescript file. The overrides are functions that take the following arguments:
+- `args` - The arguments of the operation i.e. the data that's sent from the client.
+- `context` - Context containing the `user` making the request and the `entities` object containing the entity that's being operated on.
+
+You can also import types for each of the functions you want to override from `@wasp/crud/{crud name}`. The types that are available are:
+- `GetAllQuery`
+- `GetQuery`
+- `CreateAction`
+- `UpdateAction`
+- `DeleteAction`
```

Comment:
So far we specified this stuff in special Typescript section of Docs. I am ok with mentioning it here, but I would also mention it there, just for consistency. In the future I hope we will merge it all together though.